cmd/dumpling/main.go

```diff
@@ -63,12 +65,14 @@ func init() {
 	rootCmd.PersistentFlags().IntVarP(&port, "port", "P", 4000, "TCP/IP port to connect to")
 	rootCmd.PersistentFlags().StringVarP(&password, "password", "p", "", "User password")
 	rootCmd.PersistentFlags().IntVarP(&threads, "threads", "t", 4, "Number of goroutines to use, default 4")
-	rootCmd.PersistentFlags().Uint64VarP(&fileSize, "filesize", "F", export.UnspecifiedSize, "The approximate size of output file")
+	rootCmd.PersistentFlags().Uint64VarP(&fileSize, "file-size", "F", export.UnspecifiedSize, "The approximate size of output file")
```

Comment:
refer to mydumper, seem we need to use below instead?
`-F, --chunk-filesize        Split tables into chunks of this output file size. This value is in MB`

Reply:
I think `file-size` makes more sense